assets/js/src/subscribers/importExport/import/step_data_manipulation/generate_column_selection.jsx

```diff
@@ -2,17 +2,12 @@ import jQuery from 'jquery';
 import MailPoet from 'mailpoet';
 
 export default () => {
+  const select2Config = {
+    data: window.mailpoetColumnsSelect2,
+    width: '15em',
+  };
   jQuery('select.mailpoet_subscribers_column_data_match')
-    .select2({
-      data: window.mailpoetColumnsSelect2,
-      width: '15em',
-      templateResult(item) {
```

Comment:
FYI these callbacks are no longer needed because select2 automatically looks for a `text` property, which now exists again.